diff --git a/ImNodes/ImNodes.cpp b/ImNodes/ImNodes.cpp
index 2b8cef8..eaa5a65 100644
--- a/ImNodes/ImNodes.cpp
+++ b/ImNodes/ImNodes.cpp
@@ -24,6 +24,7 @@
 #include "ImNodes.h"
 
 #include <imgui_internal.h>
+#include <cmath>
 #include <limits>
 
 namespace ImNodes
@@ -36,8 +37,8 @@ CanvasState* gCanvas = nullptr;
 
 bool operator ==(const ImVec2& a, const ImVec2& b)
 {
-    return abs(a.x - b.x) < std::numeric_limits<float>::epsilon() &&
-           abs(a.y - b.y) < std::numeric_limits<float>::epsilon();
+    return std::abs(a.x - b.x) < std::numeric_limits<float>::epsilon() &&
+           std::abs(a.y - b.y) < std::numeric_limits<float>::epsilon();
 }
 
 enum _ImNodesState
